src/releng_channel_status/releng_channel_status/templates/layout.html

```diff
@@ -0,0 +1,13 @@
+<html>
+    <head>
+        <title>
+            PAGE TITLE
```

Comment:
How about: "Nightly status"?

Reply:
Oh yeah, the current HTML is just to you see what informations we have available and to facilitate the review.